src/admin/router.ts

```diff
@@ -16,7 +17,7 @@ export const authenticateUser = async (email, password) => {
   return null;
 };
 
-export const expressAuthenticatedRouter = (adminJs: AdminJS) => {
+export const expressAuthenticatedRouter = (adminJs: AdminJS, router: Router | null = null) => {
```

Comment:
Why can not be optional and then alternatively undefined?

Reply:
`buildAuthenticatedRouter` is typed to accept `Router` or `null`. I'd have to do `router ?? null` and I don't think it's much better. With `, router: Router | null = null` you can also ignore passing 2nd param and it will set the `router` value to `null` if nothing is specified